docker-compose.m1.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
```

Comment:
The need for this dockerfile should be documented in the README. (also, is it not possible for a non-m1 user to use this same dockerfile and should we just make this the default?)

Reply:
Travis got very upset with that version of the dockerfile: https://app.travis-ci.com/github/RockefellerArchiveCenter/pictor/builds/234134522 , though looking at it again this could maybe be dealt with in the Travis configs?

(I will update the README if we don't end up collapsing these again.)

Reply:
Yeah it's odd that config isn't supported because it's been part of compose since 2.4. Let's make sure there's not something else going on here, like YAML formatting or something.

Reply:
I looked closely and I can't find anything off with the YAML formatting (and I can't reproduce this error locally). I agree that it's odd and will try to look at this again tomorrow with fresh eyes...

Reply:
I still can't figure out what's going on, so I've updated the README (and realized the README needs some work generally, so we can work on #20 next sprint).